tuf/scripts/repo.py

```diff
@@ -1141,6 +1141,10 @@ def parse_arguments():
   except (tuf.exceptions.Error) as e:
     sys.stderr.write('Error: ' + str(e) + '\n')
     sys.exit(1)
-
+  except (securesystemslib.exceptions.UnsupportedLibraryError) as e:
+    sys.stderr.write('Error: ' + str(e) + '\n')
+    sys.stderr.write('resolve this problem by:\n')
+    sys.stderr.write('sudo pip install securesystemslib[crypto,pynacl]\n')
```

Comment:
I think it's better to point the user to the [TUF installation instructions](https://github.com/theupdateframework/tuf/blob/develop/docs/INSTALLATION.rst) here.